src/docfx/lib/moniker/Moniker.cs

```diff
@@ -8,8 +8,10 @@ namespace Microsoft.Docs.Build
 {
     internal class Moniker
     {
+        [JsonProperty(PropertyName = "moniker_name")]
         public string Name { get; set; } = string.Empty;
```

Comment:
To be consistent in with data model:
```suggestion
        public string MonikerName { get; set; } = string.Empty;
```

Also use `[JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]` over `[JsonProperty]`